# 1 УРОК СЛОЖЕНИЕ И ПРОИЗВЕДЕНИЕ ВЕРОЯТНОСТЕЙ

## 2

## Задание 1

Два человека одновременно стреляют по мишени. Вероятность попадания по мишени у первого стрелка равна 0,5, у второго — 0,8. 

Какова вероятность того, что в мишени будет только одна пробоина?

Примечание: в этой и последующих задачах десятичным разделителем должна быть точка. 

In [ ]:
Подсказка

- **Что надо иметь в виду при решении?**
    - Определиться, какие ситуации здесь являются взаимоисключающими (зависимыми), а какие нет (независимыми). 
    
    Это однозначно даст вам понять, 
    
    перемножаются ли вероятности или вычитаются/складываются.
    - У задачи четыре исхода: а) не попал никто б) попали оба в) попал только первый стрелок г) попал только второй стрелок. 
        Под вопрос задачи попадают два последних варианта.

**Разбор решения**

В данной задаче вам необходимо рассчитать вероятность тех случаев, что один стрелок попадет, а другой стрелок промажет. 
Это можно сделать следующим образом:

1. Рассчитать вероятность того, что оба стрелка попадут в мишень - это будет перемножение вероятностей: $0.5 \times 0.8 = 0.4$
2. Рассчитать вероятность того, что оба стрелка промажут по мишени - это будет перемножение вероятностей промаха: $(1 - 0.5) \times (1 - 0.8) = 0.1$
3. Так как у нас суммарная вероятность всех событей равна 1, то мы можем вычесть вероятность этих двух ситуаций и получить вероятность того, что хотя бы один из стрелков попадет в мишень: $1 - 0.4 - 0.1 = 0.5$

Также можно пойти другим путем и определить вероятности событий:

1. Первый стрелок попал, а второй стрелок промахнулся: $0.5 \times (1 - 0.8) = 0.1$
2. Ситуация наоборот, первый стрелок промахнулся, а второй попал: $(1 - 0.5) \times 0.8 = 0.4$
3. Сложить эти вероятности, так как это две взаимоисключающие ситуации, которые обе складываются в событие "один стрелок попал” $0.1 + 0.4 = 0.5$

##### ОТВЕТ: 0.5 (с 8й попытки) зато сам)

## 3

## Задание 2

Трое аналитиков на экзамене независимо друг от друга решают одну и ту же задачу. 

Вероятности ее решения этими аналитиками равны 0,8, 0,7 и 0,6 соответственно. 

Найдите вероятность того, что хотя бы один аналитик решит задачу.

In [ ]:
Подсказка
Ситуация, противоположная "хоть один аналитик решил задачу" - "никто не решил задачу". 
Если сумеете найти вероятность второй ситуации - автоматически найдёте вероятность первой.

Решение задачи каждым конкретным аналитиком, событие независимое, поэтому мы можем рассчитать вероятности того, 

решит каждый аналитик задачку или нет:

$q_1 = 1 - p_1 = 1 - 0.8 = 0.2$

$q_2 = 1 - p_2 = 1 - 0.7 = 0.3$

$q_3 = 1 - p_3 = 1 - 0.6 = 0.4$

Тогда по теореме умножения вероятностей независимых событий: $p_{(0)} = q_1 \times q_2 \times q_3 = 0.2 \times 0.3 \times 0.4 = 0.024$ - вероятность того, что будет три промаха. 
    Тогда $1 - p_{(0)} = 1 - 0.024 = 0.976$ – вероятность того, что хотя бы один аналитик сможет решить задачку.

###### ОТВЕТ: 0.976 (уже со второй попытки)

# 4

## Задание 3

В первой урне находятся 10 белых и 4 черных шаров, а во второй 5 белых и 9 черных шаров. Из каждой урны вынули по шару. Какова вероятность того, что оба шара окажутся черными? В ответ запишите значение, округленное до тысячных.

Подсказки

In [ ]:
Подсказка

Что здесь все события независимы. Также не забывайте, 
что вероятность достать шарик - это нужные шарики относительно всех шариков на одну коробку.

**Разбор решения**

В этой задаче все события неависимы друг от друга, поэтому нужно просто правильно определить вероятности:

1. Вероятность вытащить черный шар из первой урны - $4/14$
2. Вероятность вытащить черный шар из второй урны - $9/14$

Так как события независимые, то нам достаточно просто перемножить полученные значения $- 4/14 \times 9/14 = 0.1836735$ и округляя до тысячных мы получаем $0.184$

##### ОТВЕТ: 0.184 (с 4 попытки) 2 попытка неправильно прочитал сколько чего. 1 раз неправильно округлил(0,1836734) не 0,183 а 1,184

# 5

## Задание 4

Вероятность хотя бы одного попадания в цель при четырех выстрелах равна 0,9984. Найдите вероятность попадания в цель при одном выстреле. 

Подсказки

In [ ]:
Подсказка

Вспомнить принцип решения второй задачи. А также понять, что вероятность попадания при каждом выстреле одинаковая, 
и эти выстрелы не зависят друг от друга.

**Разбор решения**

Пусть вероятность попадания в цель при одном выстреле будет равна - $p$. Каждый выстрел это независимое событие. Тогда, вероятность хотя бы одного попадания в цель при четырех выстрелах равна $1 - (1 - p) ^4$ (так как вероятность, что не будет ни одного попадания, равна $(1 - p)^4$). По условию это выражение равно $0,9984$. Тогда нам нужно просто решить уравнение: $1 - (1 - p) ^4 = 0,9984$

(1 - p) ^4 = 0,0016

(1 - p) = 0,2

p = 0.8




In [ ]:
не поподания = 1 - 0,9984 =  0,0016
P - вероятность попадания. 1-P вероятность промаха
(1-P)(1-P)(1-P)(1-P) = 0,0016
(1-P)**4 == 0,0016
1-P = 0,2( корень 4 степени)
P = 0.8

а обратная проверка ??

###### ОТВЕТ: 0.8 (11 попыток) схалтурил - загуглил

# 6

## Задание 5

Из колоды в 36 карт наудачу извлекается одна карта. События:

A = {вынутая карта – туз}

В = {вынутая карта чёрной масти}

F = {вынутая карта – фигура, т.е. является валетом, дамой, королём или тузом}.

Выберите все пары зависимых событий:

Подсказки

Это когда появление одного события меняет вероятность другого события. 

Здесь есть события, попадающие именно во вторую категорию.

##### ОТВЕТ: A и F  (с 1 попытки)

##### из разбора


События B и F будут независимыми, так как вы можете вытянуть любую карту черной масти, а вот окажется ли она "фигурой" — ещё не факт

События A и B также будут независимыми, так как вы можете вытянуть туз и красной, и черной масти.

Последняя пара событий как раз будет являться зависимой — ведь если вы вытяните туз, то вероятность вытянуть туз в событии F изменится, ведь тузов в колоде станет меньше

# 7

## Задание 6

В конверте находится 10 лотерейных билетов, среди которых 3 выигрышных. Из конверта последовательно извлекаются билеты. Найти вероятность того, что три первых лотерейных билета будут выигрышными. Ответ округлите до десятитысячных. В качестве десятичного разделителя используйте точку.

Подсказки

In [ ]:
Всякий раз, когда мы берём билет, общее количество билетов уменьшается - равно как и количество выигрышных билетов. 

Соответственно, 

после взятия каждого выигрышного билета вероятность вытащить ещё один меняется.

**Разбор решения**

Вероятность того, что первый вытащенный конверт будет выигрышный - $3/10$, вероятность того, что и второй конверт будет выигрышный - $2/9$, если один выигрышный уже достали, третий - $1/8$. По правилу умножения вероятностей получаем, что

Вероятность вытащить три выигрышных билета $= 3/10 * 2/9 * 1/8 ≈ 0.0083$

###### ОТВЕТ: 0.0083 (с 1 попытки и без подсказок)

# 8

## Задание 7

В трех углах равностороннего треугольника находится по роботу. Каждый из роботов начинает двигаться в случайно выбранный угол по стороне треугольника. Какова вероятность того, что ни один из роботов не столкнется с другим роботом?

Ответ округлите до сотых. В качестве десятичного разделителя используйте точку.

Подсказки

- **Какие события удовлетворяют нашему условию?**
    
    Чтобы никто друг с другом не столкнулся, все роботы должны идти в одном направлении - то есть либо все идут по часовой стрелке, либо против неё.
    
- **А сколько всего событий?**
    
    Так как вариантов движения у каждого робота два, а всего роботов три, то их количество равно $2^3$.

**Разбор решения**

Роботы не столкнутся только в том случае, если не пойдут навстречу друг другу, то есть, если роботы расположены в вершинах A, B, C, то схема переходов должна быть следующая

$**A -> B, B -> C, C -> A**$

Или же в обратном порядке:

$**A -> C, C -> B, B -> A**$

Всего количество способов для роботов $= 2^3 = 8$. Нам подходят только $2 => ответ = 2/8 = 0.25$

In [ ]:
ОТВЕТ: 0.25 (5 попыток) схалтурил - загуглил

# 9

## Задание 8

Брошены две игральные кости. Есть следующие события:

А = {выпадение шестерки на первой кости}
В = {сумма выпавших очков равна 7}
С = {выпадение тройки на второй кости}

Какие пары событий являются независимыми? 

Подсказки

- **Что такое "зависимые события"?**
    
    Это когда появление одного события меняет вероятность другого события. 
    
- **А как это можно проверить?**
    
    Пользуясь формулой $P(A \cap B) = P(A)P(B)$, где $\cap$ обозначает символ "И". Человеческим языком: если произведение вероятностей двух событий равно вероятности одновременного наступления этих событий, то они **независимы**.
    
    Попробуйте посчитать вероятность каждого из трёх вариантов, а затем посчитать, с какой вероятностью произойдут два из них одновременно. Есть ли случай, где эти варианты различаются? Или все одинаковы?

###### ОТВЕТ: А и С В и С (7 попытки) тупо перебором(())

##### из разбора

Какие пары событий независимы:

А={выпадение шестерки на первой кости}

В={сумма выпавших очков равна 7}

С={выпадение тройки на второй кости}

Довольно очевидно, что A и C друг от друга не зависят - ведь бросок одной кости никак не меняет результат на другой. Но как быть с A и B, а также B и C? Проверим это через формулу независимости событий:

P(A∩B)=P(A)P(B)

Символ ∩ обозначает здесь "И".

P(A)=1/6 (так как всего граней шесть, а нас интересует всего одно значение)

P(B)=6/36=1/6 (всего комбинаций двух кубиков 36, а условию соответствуют варианты 6 и 1, 5 и 2, 4 и 3, 3 и 4, 2 и 5, 1 и 6)

P(A∩B)=1/36 (если выпала шестерка, то событие B произойдёт только для одной комбинации из 36 - 6 и 1)

Итого: 1/36=1/6∗1/6, что соответствует независимости событий. То же самое справедливо для события C.

Значит, все эти события независимы.

NB! Если бы в качестве события B у нас была любая другая сумма, то тогда A и C/A и B были бы зависимыми. Это связано с тем, что в этом случае количество возможных значений на костях стало бы ограниченным.

Например, сумма, равная двум, может быть выкинута только двумя единицами. Соответственно, наши текущие события A и C исключали бы событие B полностью. Попробуйте также проверить другие варианты - например, 8.

# 10

## Задание 9

Бросаются 2 игральных кубика. Определите вероятность того, что произведение числа очков на обоих кубиках не превосходит 10.

В ответ запишите число округленное до тысячных. В качестве десятичного разделителя используйте точку.

Подсказки

- **Сколько тут всего возможных комбинаций?**
    
    Так как граней у нас 6, а бросков 2, то общее число комбинаций равно $6^2$.
    
- **А что делать дальше?**
    
    Надо подсчитать все возможные варианты произведений очков на костях, которые дают число меньше либо равное десяти, сложить и поделить на общее число событий. Для удобства можете считать одно из значений фиксированным (например, **1*1, 1*2, 1*3, 1*4, 1*5 и 1*6** все меньше 10).
    
    **NB!** События *"на первой кости выпало 1, на второй 6"* и *"на первой кости выпало 6, на второй 1" -* это **два разных события**. Не забудьте учесть оба.

In [9]:
count = 0
for i in range(1,7):
    for j in range(1,7):
        if i*j <= 10:            
            count += 1
            print(i,j, count)
print(count)


1 1 1
1 2 2
1 3 3
1 4 4
1 5 5
1 6 6
2 1 7
2 2 8
2 3 9
2 4 10
2 5 11
3 1 12
3 2 13
3 3 14
4 1 15
4 2 16
5 1 17
5 2 18
6 1 19
19


In [10]:
19/36 

0.5277777777777778

###### ОТВЕТ: 0.528 (с 5 попытки)

**Разбор решения**

Всего способов подкинуть два кубика -- 36. Нас устраивают из них следующие варианты **(произведение которых не больше 10)**:

(1, 1) (1, 2) ... (1, 6)

(2, 1) (2, 2) ... (2, 5)

(3, 1) (3, 2) (3, 3)

(4, 1) (4, 2)

(5, 1) (5, 2)

(6, 1)

Всего таких вариантов $19 =>$  ответ $19/36$

In [ ]:
количество попыток при ответе на 1 урок
1  (с 8й попытки) зато сам)
2  (уже со 2й попытки)
3  (с 4 попытки) 2 попытка неправильно прочитал сколько чего. 1 раз неправильно округлил
4  (11 попыток) схалтурил - загуглил
5  (с 1 попытки)
6  (с 1 попытки и без подсказок)
7  (5 попыток) схалтурил - загуглил
8  (7 попытки) тупо перебором(())
9  (с 5 попытки хорошо Python помог)